docs/src/get-started.md

```diff
@@ -0,0 +1,191 @@
+
+Available inference methods include  Importance Sampling (IS), Sequential Monte Carlo (SMC), Particle Gibbs (PG), Hamiltonian Monte Carlo (HMC), Hamiltonian Monte Carlo with Dual Averaging (HMCDA) and The No-U-Turn Sampler (NUTS).
+
+### Simple Gaussian demo
+
+Below is a simple Gaussian demo illustrate the basic usage of Turing.jl
+
+```julia
+# Define a simple Normal model with unknown mean and variance.
+@model gdemo(x) = begin
+  s ~ InverseGamma(2,3)
+  m ~ Normal(0,sqrt(s))
+  x[1] ~ Normal(m, sqrt(s))
+  x[2] ~ Normal(m, sqrt(s))
+  return s, m
+end
+```
+
+Note: for the interests of sanity check, some analytical results on the expectation of samples of this model are E[s] = 49/24 and E[m] = 7/6.
+
+Inference methods are functions which take the probabilistic program as one of the arguments.
+
+```julia
+#  Run sampler, collect results
+c1 = sample(gdemo([1.5, 2]), SMC(1000))
+c2 = sample(gdemo([1.5, 2]), PG(10,1000))
+c3 = sample(gdemo([1.5, 2]), HMC(1000, 0.1, 5))
+c4 = sample(gdemo([1.5, 2]), Gibbs(1000, PG(10, 2, :m), HMC(2, 0.1, 5, :s)))
+c5 = sample(gdemo([1.5, 2]), HMCDA(1000, 0.15, 0.65))
+c6 = sample(gdemo([1.5, 2]), NUTS(1000,  0.65))
+
+# Summarise results
+Mamba.describe(c3)
+
+# Plot results
+p = Mamba.plot(c3)
+Mamba.draw(p, fmt=:pdf, filename="gdemo-plot.pdf")
+```
+
+The arguments for each sampler are
+
+* SMC: number of particles
+* PG: number of particles, number of iterations
+* HMC: number of samples, leapfrog step size, leapfrog step numbers
+* Gibbs: number of samples, component sampler 1, component sampler 2, ...
+* HMCDA: number of samples, total leapfrog length, target accept ratio
+* NUTS: number of samples, target accept ratio
+
+For detailed information please check Turing.jl's [APIs](https://github.com/yebai/Turing.jl/wiki/APIs).
+
+### Modelling syntax explained
+
+Models are wrapped by `@model` with a normal function definition syntax, i.e.
+
+```julia
+@model model_name(arg_1, arg_2) = begin
+  ...
+end
+```
+
+This syntax defines a model which can take data as input to generate a posterior evaluator. The data can be provided either using the same function signature defined, or by using a dictionary containing each argument and its value as pairs, i.e.
```

Comment:
`posterior evaluator`: what do you mean by this? Maybe define this more accurately.

Reply:
Since I'm a bit of a probabilistic neophyte, I'm a bit curious on this as well. I think intuitively I would think of it as (depending on context) a something to be trained by feeding it data to update your posterior, or as something to predict values using an extant posterior.

Is that close?